### PR TITLE
(ztp.ls.lsst.org) EL9 Redeployment.

### PR DESCRIPTION
**Pull request by dtapiacl**
IHS-4998

### CODE REVIEW

File: hieradata/node/it-ztp.ls.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    content:
+      connection:
+        id: "ens192"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "ens192"
+      ethernet: {}
+      ipv4:
+        address1: "10.49.9.253/24,10.49.9.254"
+        dns: "139.229.135.53;139.229.135.54;"
```
Comment (jhoblitt) [inline hieradata/node/it-ztp.ls.lsst.org.yaml]: We should set all 3 forward resolvers.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Can we rename this host to just `ztp`?

File: hieradata/node/it-ztp.ls.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/it-ztp.ls.lsst.org.yaml]: Included missing resolver.

**Comment by dtapiacl** [comment]
> Can we rename this host to just `ztp`?

I don't see why not, host was renamed to ztp.ls.lsst.org.

**Review by jhoblitt — APPROVED** [review]
LGTM.  Thank you for changing the hostname.

**Pull request merged**